project/Makefile

```
@@ -90,6 +90,8 @@ django-checks:
 ##
 
 ../var/requirements_frozen.txt:
+	# Pin Django directly to LTS version provided by the test matrix
+	echo 'Django~=${DJANGO_VERSION}' >> requirements.txt
```

We could just add the `.0` here instead of in the matrix. E.g.

```suggestion
	echo 'Django~=${DJANGO_VERSION}.0' >> requirements.txt
```

Looks a lot cleaner and nicer in the matrix and other checks, imho,.

---

I thought about that but didn't. But I think I agree its cleaner to add here